lib/ap/plan9/malloc.c

```diff
@@ -74,7 +74,7 @@ malloc(size_t size)
 	if(pow < CUTOFF) {
 		n = (CUTOFF-pow)+2;
 		bp = sbrk(size*n);
-		if((intptr_t)bp == -1)
+		if((long)bp == -1)
```

Comment:
I tried with 64 bit integers, little tests programs seems to be ok, getenv keeps no working and I remember some rare issues in binutils/gawk, but I think was a wrong convert process. Ok, I'll update malloc to 64.